### DISCUSSION TITLE
Practica JS

### DISCUSSION THREAD

**Discussion by ArmandoRuiz13** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#436] [Q&A]
Estoy teniendo problema para acceder a el tipo de pokemon, alguien podria ayudarme para llegar a la ruta? , a veces tiene [2] tipos y a veces una.

**Comment by alanglez45** [comment]
Si te fijas bien, contiene un arreglo ya sea con 1 o dos posiciones entonces puedes jalar ese arreglo y si quieres lo iteras con un for loop y ya extraes cada tipo, igual puedes aplicarlo para las estadisticas y/o movimientos

**Reply by ArmandoRuiz13** [reply]
Tenemos que poner todos los movimientos y tipos?

**Reply by alanglez45** [reply]
No dijeron pero igual es tu app, ya es a consideración tuya. Por los tipos no hay problema tienen como maximo 2. Los movimientos pues si son muchos.
